game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtils.java

```diff
@@ -429,18 +431,41 @@ private static double findWaterValue(
     return capitalOrFactoryValue / 100 + nearbyLandValue / 10;
   }
 
+  // 1700 oldT=4906 newT=227 newT2=166 newT3=152
```

Comment:
Is this comment intended to be included?

Reply:
No, removed. Thanks!